src/Illuminate/Foundation/Providers/FoundationServiceProvider.php

```diff
@@ -135,8 +137,6 @@ public function registerDumper()
      * Register the "validate" macro on the request.
      *
      * @return void
-     *
-     * @throws \Illuminate\Validation\ValidationException
```

Comment:
This method just register macro methods for the Request class. Validation exceptions are only thrown when `$request->validate()` or `$request->validateWithBag()` are called and failed.